src/Components/header.html

```diff
@@ -1,10 +1,20 @@
-<header id="header">
+<header id="header" class="flex items-center  justify-between py-[24px]">
     <div class="logo">
-        <img src="" alt="">
+        <img src="/src/Assets/media/Ojemba-logo.svg" alt="lll">
     </div>
     <nav>
-        <ul>
-            <li><a href="#">Home</a></li>
+        <ul class="flex font-robotoBold text-white gap-[32px] text-[18px]" >
+            <li><a href="#">Services</a></li>
+            <li><a href="#">Über uns</a></li>
+            <li><a href="#">Darum wir</a></li>
+            <li><a href="#">Team</a></li>
+            <li><a href="#">Über Ruanda</a></li>
+            <li><a href="#">FAQ</a></li>
+            <li><a href="#">Kontakt</a></li>
+            <li class="flex flex-col">
+                <a href="#">DE</a>
+                <a href="#">EN</a>
+            </li>
         </ul>
     </nav>
```

Comment:
Hey @Kevinemug, Please the wrapper on the header are not conform to the one on the figma design Please make sure you refer to Figma design for it to conform to design QA
![header-review](https://github.com/Bateyjosue/ojemba-team-project/assets/52311487/0eb4673b-c7d6-4f76-9272-560baf1db925)

![header review 1](https://github.com/Bateyjosue/ojemba-team-project/assets/52311487/a7c69902-c18b-4d57-81f9-8ab874270b96)

Reply:
Thank you @Bateyjosue  I'm going to work on it